app/views/tasks/show.html.slim

```diff
@@ -1,632 +1,621 @@
 
-  /.button-box
-    .button-description
-      = "Derivate, translate or port this Exercise by duplicating it. The duplicate will be owned by you."
-    .wrapper
-      = link_to t('exercises.all.duplicate'), duplicate_exercise_path(@task), class:'btn btn-default'
+      /.button-box
+        .button-description
+          = "Derivate, translate or port this Exercise by duplicating it. The duplicate will be owned by you."
+        .wrapper
+          = link_to t('exercises.all.duplicate'), duplicate_exercise_path(@task), class:'btn btn-light'
 
-/.related-panel
-  = button_tag type: 'button', class: 'full-window-button related-button' do
-    => t('exercises.all.related_exercises')
-    span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
-    span.fa.fa-caret-down.my-caret
-/div class=("related-box with-borders") data-exercise=@task.id
-  .slide-left.inactive
-    = button_tag do
-      i.fa.fa-caret-left
-  .content
-  .slide-right.inactive
-    = button_tag do
-      i.fa.fa-caret-right
-.expandable-panel.with-borders
-  = button_tag type: 'button', class: 'full-window-button show-comment-button', data: {task_id: @task.id} do
-    => t('comments.show_comments')
-    span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
-    span.fa.fa-caret-down.my-caret
-  .comment-box data-task=@task.id
-    = form_for :comment, url: task_comments_path(@task), remote: true do |f|
-      .write-comment
-        .input-group
-          = f.label_tag 'new_comment_label', 'New Comment', class: 'input-group-addon'
-          = f.text_field :text, class: 'form-control'
-          span.input-group-btn
-            = f.submit class: 'btn btn-default'
-    .comment-body
+    /.related-panel
+      = button_tag type: 'button', class: 'full-window-button related-button' do
+        = t('exercises.all.related_exercises')+ ' '
+        span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
+        span.fa.fa-caret-down.my-caret
+    /div class=("related-box with-borders") data-exercise=@task.id
+      .slide-left.inactive
+        = button_tag do
+          i.fa.fa-caret-left
+      .content
+      .slide-right.inactive
+        = button_tag do
+          i.fa.fa-caret-right
+    .expandable-panel.with-borders
+      = button_tag type: 'button', class: 'full-window-button show-comment-button', data: {task_id: @task.id} do
+        => t('comments.show_comments')
+        span class="fa fa-circle-o-notch fa-spin wait" style="display:none;margin-right:5px;"
+        span.fa.fa-caret-down.my-caret
+      .comment-box data-task=@task.id
+        = form_for :comment, url: task_comments_path(@task), remote: true do |f|
+          .write-comment
+            .input-group
+              = f.label 'new_comment_label', 'New Comment', class: 'btn btn-light disabled input-group-addon'
```

Comment:
Same comment as above: Is this actually a disabled button, or rather `input-group-text`?